src/main/java/Domain/Controllers/TeamController.java

```diff
@@ -388,4 +406,74 @@ private static boolean roleStillExists(Role role){
             return true;
         return false;
     }
+
+    /**
+     * Removes the owner form the team he owned, the method checks the owner to remove is one of the owners of the team the method recieved
+     * @param username to remove
+     * @param team who owned for removal
+     * @param owner who made the action
+     * @return
+     * @throws UserNotFoundException
+     * @throws NotATeamOwner
+     */
+    public static boolean removeTeamOwner(String username, Team team, TeamOwner owner) throws Exception {
+        List<TeamOwner> teamOwners = team.getTeamOwners();
```

Comment:
why is this function receives a team and a teamOwner? why doesn't it asks for the team by choice and owner by choice?

Reply:
מה שכתבת מתבצע בקונטרולר שבסרוויס שקורה לפונקציה הזאת שנמצאת בדומיין

Reply:
יואו סליחה אתה צודק, לא שמתי לב שזה קונטרולר בשכבת הדומיין